src/Theme/index.js

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { StyleSheet } from "react-native";
+import defaultBranding from "../branding_defaults";
+
+export const ThemeContext = React.createContext({
+  theme: {}
+});
+
+export class Provider extends React.Component {
+  themeConfigs = {};
+  branding = defaultBranding;
+
+  constructor(props) {
+    super(props);
+    if (props.branding) {
+      this.branding = deepMerge(this.branding, props.branding);
```

Comment:
There is no deepMerge func.

Reply:
HA! I guess I should also add some tests for this